lib/logitech_receiver/receiver.py

```diff
@@ -311,13 +311,16 @@ def __hash__(self):
 	__bool__ = __nonzero__ = lambda self: self.wpid is not None and self.number in self.receiver
 
 	def __str__(self):
-		return '<PairedDevice(%d,%s,%s)>' % (self.number, self.wpid, self.codename or '?')
+		return '<PairedDevice(%d,%s,%s,%s)>' % (self.number, self.wpid, self.codename or '?', self.serial)
 	__unicode__ = __repr__ = __str__
 
 #
 #
 #
 
+# Lightspeed receivers
+lightspeed_receiver_product_ids = { 'c539', 'c53a', 'c53f' }
```

Comment:
I was wondering whether this list is a kind of duplicate of the information in `lib/logitech_receiver/base_usb.py`? Would it be an practical idea to create already tuples in base_url.py?

Reply:
Hmm.  Maybe. Let me look.

Reply:
Because when I first added support for Lightspeed receivers I didn't have time to implement it properly.

Reply:
It might be OK to do this, but I'm not sure that it is a good idea to add stuff into base_usb.py.
@FFY00 Any comments?

Reply:
See my comment in f4557233f1e0cd50986ae1eac44c882cfe255879.

Reply:
OK.  I'll add this information in base_usb.py.
@FFY00 Determination of whether a device can unpair should also be upgraded.  Right now, the determination is incorrect for at least c534 receivers.   Can this information be obtained from Logitech?  Also, c534 receivers can pair by replacing existing devices.  I've added this capability just for them, but maybe other receivers can also do this.  Can this information be obtained from Logitech?

Reply:
The information in base_usb.py is a four-tuple, and code that uses the information depends on that.  I'm going to change this to a dictionary so that later additions are much easier.